### DISCUSSION TITLE
经过hysteria的所有流量是否都是加密的

### DISCUSSION THREAD

**Discussion by ledouz94** [apernet/hysteria#690] [Q&A] [answered]
我想通过tcp relay的功能来代理我的后端网站，remote地址端口可否配置成http协议的80端口（而不是https）？通过hysteria传输的流量是否都已经加密过的？

**Comment by tobyxdd** [comment, marked as answer]
当然，QUIC 流量都必须是 TLS 的

**Reply by thisisjacky6** [reply]
tls使用哪个证书，在服务器端的acme下只写了域名，并未写私钥地址如何加密

**Reply by ans-** [reply]
默认自动生成letsencrypt证书
